packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -1,10 +1,11 @@
 import {
     getResolvedShapeDescriptors,
+    createConnectorForGlobalObject,
     createMembraneMarshall,
-    marshallSourceTextInStrictMode,
     linkIntrinsics,
     DistortionCallback,
     SupportFlagsObject,
+    validateRequiredObjects,
```

Comment:
I like `validateRequiredObjects`. Reminds me of code I was just looking at in Node.js internals.